pages/common/wondershaper.md

```diff
@@ -0,0 +1,28 @@
+# wondershaper
+
+> Wonder Shaper is a script that allows the user to limit the bandwidth of one or more network adapters.
+> More information: <https://github.com/magnific0/wondershaper#usage>.
+
+- Print the [h]elp:
+
+`wondershaper -h`
```

Comment:
Does this have long versions of these flags such as --help instead?

Reply:
no, it doesn't have